### PR TITLE
Week 1-2 progress update: Sukanta Roy

### PR DESCRIPTION
**Pull request by sukanta-27**
- [x] Add week 1 and 2 progress report
- [x] Update project plan

I have added the progress report and updated my project plan. Also in the project plan, I have removed some redundant `Code Review and Fix` entries. @xpdavid @bqnguyen94 Please have a look.

### CODE REVIEW

**Review by bqnguyen94 — COMMENTED** [review]
I see that you've decided to do backend and frontend parts of `MCQ questions` back-to-back as discussed with @xpdavid instead of splitting them up? I also prefer this approach for myself, though it often results in gigantic PRs that may take days to merge, so try to submit your PR and let us know soon as possible!

Also, don't forget let us know in the next retrospect how this change of approach works out for you. Good luck :)

**Comment by sukanta-27** [comment]
@bqnguyen94 Thanks a lot.

>Also, don't forget let us know in the next retrospect how this change of approach works out for you. Good luck :)

 Ofcourse will keep you guys updated. :smile:

File: Sukanta_Roy/progress.md [hunk 1]
```diff
@@ -19,58 +19,68 @@ Overall Direction:
...
+
+This week the plan was to complete the backend of MCQ weights feature, and I edited `FeedbackMcqQuestionDetails.java` accordingly, but
+I forgot to take into account about MCQ `other` option and how weight feature should behave in case of the `Other` option. After discussing with my mentors and Prof. Damith, we came to the conclusion that, at this point, we will add `weight` feature to `other` option too, giving the instructor full flexibility of the feature. We also discussed the best possible way to store the necessary fields in the datastore. In coming weeks, I will have to change my initial implementation of the weight feature to add the feature for the `Other` option. 
+
 Week 2 Plan:
 **Final Semester Exams**
 - [ ] Finalize the sample question list for the new question template
 
+Week 2 Work:
+- [x] Made 3 sample templates to choose from, for the `Add new template` feature.
```
Comment (xpdavid) [inline Sukanta_Roy/progress.md]: Include the link to the issue?

**Review by xpdavid — APPROVED** [review]
LGTM. We are expecting a PR coming this week for your back-end implementation. Submit ASAP so we can check if you are in the right track. Good luck :)

@bqnguyen94 anything to add on?

**Review by bqnguyen94 — APPROVED** [review]

File: Sukanta_Roy/progress.md (hunk 1 above)
Comment (sukanta-27) [reply Sukanta_Roy/progress.md]: Oops! forgot to include the link after creating the issue. Doing that.

**Comment by sukanta-27** [comment]
@bqnguyen94 Added the issue link. Ready for merge.

**Comment by sukanta-27** [comment]
>LGTM. We are expecting a PR coming this week for your back-end implementation. Submit ASAP so we can check if you are in the right track. Good luck :)

@xpdavid Thank you. Will submit it ASAP.

**Pull request merged**